pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -26,6 +26,7 @@ type PerconaXtraDBClusterSpec struct {
 	Backup                *PXCScheduledBackup                  `json:"backup,omitempty"`
 	UpdateStrategy        appsv1.StatefulSetUpdateStrategyType `json:"updateStrategy,omitempty"`
 	AllowUnsafeConfig     bool                                 `json:"allowUnsafeConfigurations,omitempty"`
+	SecurityContext       *corev1.PodSecurityContext           `json:"securityContext,omitempty"`
```

Comment:
I think it should be possible to set on the level of PXC, ProxySQL, PMM, and storage level of Backup